gpflow/conditionals/conditionals.py

```diff
@@ -126,9 +126,12 @@ def _conditional(
         - mean:     [N, R]
         - variance: [N, R] (full_cov = False), [R, N, N] (full_cov = True)
     """
-    Kmm = kernel(X) + eye(tf.shape(X)[-2], value=default_jitter(), dtype=X.dtype)  # [..., M, M]
-    Kmn = kernel(X, Xnew)  # [M, ..., N]
-    Knn = kernel(Xnew, full_cov=full_cov)  # [..., N] (full_cov = False) or [..., N, N] (True)
-    mean, var = base_conditional(Kmn, Kmm, Knn, f, full_cov=full_cov, q_sqrt=q_sqrt, white=white)
-
-    return mean, var  # [N, R], [N, R] or [R, N, N]
+    posterior = VGPPosterior(
+        kernel=kernel,
+        X=X,
+        q_mu=f,
+        q_sqrt=q_sqrt,
+        white=white,
+        precompute_cache=None,
+    )
+    return posterior.fused_predict_f(Xnew, full_cov=full_cov, full_output_cov=full_output_cov)
```

Comment:
It's probably not a big deal, but does this mean that `VGP_deprecated` indirectly depends on `VGP_with_posterior` (via the conditional/dispatch) which itself inherits from `VGP_deprecated`?

Reply:
> It's probably not a big deal, but does this mean that `VGP_deprecated` indirectly depends on `VGP_with_posterior` (via the conditional/dispatch) which itself inherits from `VGP_deprecated`?

Yes. This behaviour is consistent with the behaviour in the SVGP, defined above. I've just copied the software architecture.